### PR TITLE
add comma to long sentence in docs

### PR DESCRIPTION
**Pull request by pjfanning**
based on #451

### CODE REVIEW

File: docs/src/main/paradox/typed/fault-tolerance.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ It is possible to override this so that child actors are not influenced when the
...
-when parent is restarted the @apidoc[supervise](typed.*.Behaviors$) {scala="#supervise[T](wrapped:org.apache.pekko.actor.typed.Behavior[T]):org.apache.pekko.actor.typed.scaladsl.Behaviors.Supervise[T]" java="#supervise(org.apache.pekko.actor.typed.Behavior)"} should be placed inside the @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:org.apache.pekko.actor.typed.scaladsl.ActorContext[T]=%3Eorg.apache.pekko.actor.typed.Behavior[T]):org.apache.pekko.actor.typed.Behavior[T]" java="#setup(org.apache.pekko.japi.function.Function)"} and using
+when parent is restarted, the @apidoc[supervise](typed.*.Behaviors$) {scala="#supervise[T](wrapped:org.apache.pekko.actor.typed.Behavior[T]):org.apache.pekko.actor.typed.scaladsl.Behaviors.Supervise[T]" java="#supervise(org.apache.pekko.actor.typed.Behavior)"} should be placed inside the @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:org.apache.pekko.actor.typed.scaladsl.ActorContext[T]=%3Eorg.apache.pekko.actor.typed.Behavior[T]):org.apache.pekko.actor.typed.Behavior[T]" java="#setup(org.apache.pekko.japi.function.Function)"} and using
```
Comment (nvollmar) [inline docs/src/main/paradox/typed/fault-tolerance.md:120]: should be `supervisee`

Comment (pjfanning) [reply docs/src/main/paradox/typed/fault-tolerance.md:120]: @nvollmar do you mean that `@apidoc[supervise]` should be `@apidoc[supervisee]` ? If so, I can add that to this PR.

Comment (nvollmar) [reply docs/src/main/paradox/typed/fault-tolerance.md:120]: @pjfanning yes, the passiv would be supervisee

Comment (pjfanning) [reply docs/src/main/paradox/typed/fault-tolerance.md]: change made

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by gmethvin — APPROVED** [review]

**Pull request merged**